transcom/mymove
# B:19826 - Rename Transportation Ordering Officer to Task Ordering Officer

## [B-18826](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-19826)

## Summary

Renamed Transportation Ordering Officer to Task Ordering Officer

Is there anything you would like reviewers to give additional scrutiny?

[this article](tbd) explains more about the approach used.

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Access office, admin, and swagger pages
2. Make sure you are able to create a TOO (Task Ordering Officer)
3. Make sure you can Request Task Ordering Officer Role
4. Search for Transportation Ordering Officer to make sure it does not exist

### Frontend


### Backend


### Database

#### Any new migrations/schema changes:


## Screenshots
